vies/locale/nl/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,40 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-08-04 14:39+0200\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Diederik van der Boor <vdboor@edoburu.nl>\n"
+"Language-Team: \n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: __init__.py:111
+#, python-format
+msgid "%s is not a valid ISO_3166-1 country code."
+msgstr "%s is geen geldige ISO_3166-1 landcode."
+
+#: __init__.py:114
+#, python-format
+msgid "%s is not a VIES member country."
+msgstr "%s is niet een VIES lidstaat."
```

Comment:
"%s is geen VIES-lidstaat", 

zie ook http://ans.ruhosting.nl/e-ans/29/04/02/body.html en http://www.spatiegebruik.nl/regels.html